kvm-libvirt/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -1,5 +1,5 @@
 module "bootkube" {
-  source = "git::https://github.com/kinvolk/terraform-render-bootkube//?ref=7e237ffa21fd85f76ddf2a215073aa7cd6ef2476"
+  source = "git::https://github.com/kinvolk/terraform-render-bootkube//?ref=aec4326328bb7bdbd132a6e0b9c202e40f1ec04c"
```

Comment:
While other modules use `github.com/kinvolk/terraform-render-bootkube` as the source, this one uses `git::https://github.com/kinvolk/terraform-render-bootkube`. 

This might be out of scope in this PR, but would you like to make this consistent across the codebase? Either approach should be fine I believe and a follow up PR would probably be better (in the interest of getting this one merged faster :slightly_smiling_face: )

This is only a suggestion.